modules/ROOT/pages/indexes-for-vector-search.adoc

```diff
@@ -301,12 +316,12 @@ You can also set a vector property on a node using the xref:clauses/set.adoc#set
 ----
 MATCH (node:Node {id: $id})
 SET node.propertyKey = $vector
-RETURN node;
+RETURN node
 ----
 
 However, Cypher stores the provided `LIST<FLOAT>` as a primitive array of {ieee-754} _double_ precision values.
 As a result, it takes up approximately twice as much space and it's not recommended for properties that are used in a vector index.
-Existing properties can be re-set using `db.create.setVectorProperty` to minimize store size, but there is a cost in the transaction log size until they are rotated away.
+Existing properties can be re-set using `db.create.setNodeVectorProperty` to minimize store size, but there is a cost in the transaction log size until they are rotated away.
```

Comment:
This might be important to still note in some way, shape or form, as people who are using the procedure would be concerned with space; and then may be quite surprised to see that (at least for a while) the total disk spaced used increases by quite a bit.

Reply:
If you have an alternative suggestion or rough idea which may be a better approach to express this, I'd be grateful, as I agree it doesn't sit right in its current state.

Reply:
I added it as part of the text and removed it here. But if you think it's not visible enough, I can take another look.